python/pyarrow/_flight.pyx

```diff
@@ -950,7 +1005,9 @@ cdef class FlightInfo(_Weakrefable):
                 f"descriptor={self.descriptor} "
                 f"endpoints={self.endpoints} "
                 f"total_records={self.total_records} "
-                f"total_bytes={self.total_bytes}>")
+                f"total_bytes={self.total_bytes} "
+                f"ordered={'true' if self.ordered else 'false'} "
```

Comment:
This matches the C++ implementation of `ToString`. In Python, you would expect `True` or `False` though. Happy to change this to
```suggestion
                f"ordered={self.ordered} "
```

Reply:
I think that we can use `True`/`False` in Python.